packages/manager/apps/web/client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -368,14 +368,12 @@
                         data-term="{{:: 'hosting_dashboard_service_cdn' | translate }}"
                     >
                         <oui-tile-description>
-                            <span
-                                data-ng-if=""
-                            >
+                            <span data-ng-if="">
```

Comment:
Not yours but this element could be removed
```suggestion
```

Reply:
element is removed